packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -340,10 +288,10 @@ class Actions extends InheritedWidget {
       // Will only get here if nullOk is true.
       return false;
     }
-
-    // Invoke the action we found using the dispatcher from the Actions Element
-    // we found, using the given focus node.
-    return _findDispatcher(actionsElement).invokeAction(action, intent, focusNode: focusNode);
+    // Invoke the action we found using the dispatcher from the Actions we
+    // found, using the given focus node. Or null, if nullOk is true, and we
+    // didn't find something.
+    return actions?.dispatcher?.invokeAction(action, intent, focusNode: focusNode);
```

Comment:
If actions or dispatcher is null then we're returning null from a bool function. Maybe that's not what you want?

Reply:
Good point. Fixed.